pages/saved.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Image from 'next/image'
+import { FaArrowRight } from "react-icons/fa";
+import Link from 'next/link';
+
+const saved = () => {
```

Comment:
What do you think of using it as a component, not as a function? As it only returns the JSX.

```suggestion
const Saved = () => {
```